website/source/docs/upgrade-specific.html.md

```diff
@@ -15,6 +15,33 @@ This page is used to document those details separately from the standard
 upgrade flow.
 
 
+## Consul 1.7.0
+
+Consul 1.7.0 contains two major changes that may impact upgrades: 
+[stricter JSON decoding](#stricter-json-decoding) and [modified DNS outputs](#dns-output-changes)
+
+### Stricter JSON Decoding
+
+The HTTP API will now return 400 status codes with a textual error when unknown fields
+are present in the payload of a request. Previously, Consul would simply ignore the
+unknown fields. You will need to ensure that your API usage only uses supported
+fields which are those documented in the example payloads in the API documentation.
+
+### DNS Output Changes
+
+For both of the output changes, care should be take to ensure your applications utilizing
+Consul DNS can cope with the changes before upgrading.
+
+#### PTR Record Queries
+
+Consul will now return the canonical service name in response to PTR queries. For OSS users the
+change is that the datacenter will be present where it was not before. For Consul Enterprise
+users, both the datacenter and the services namespace will be present.
```

Comment:
Do we want to link out to the namespace DNS docs here?

Reply:
Do we have namespace DNS specific docs?

Reply:
We don't document PTR rr responses anywhere. We probably should at some point but for this already huge PR, probably not.